friendsofsymfony/rest-bundle/2.2/etc/packages/fos_rest.yaml

```diff
@@ -0,0 +1,17 @@
+# Read the documentation: https://symfony.com/doc/master/bundles/FOSRestBundle/index.html
+fos_rest: ~
+# Uncomment to activate features
```

Comment:
only this line which does not add any value IMHO (and the extra blank line as well)